tests/unit/fem/test_3d_bilininteg.cpp

```diff
@@ -870,6 +896,31 @@ TEST_CASE("3D Bilinear Vector Mass Integrators",
                   delete blfT;
                   delete diff;
                }
+               SECTION("With Scalar Coefficient (VectorFE)")
+               {
+                  MixedBilinearForm blf(&fespace_nd, &fespace_rt);
+                  blf.AddDomainIntegrator(new VectorFEMassIntegrator(q3_coef));
+                  blf.Assemble();
+                  blf.Finalize();
+
+                  blf.Mult(f_nd,tmp_rt); g_rt = 0.0;
+                  PCG(m_rt, s_rt, tmp_rt, g_rt, 0, 200, cg_rtol * cg_rtol, 0.0);
+
+                  REQUIRE( g_rt.ComputeL2Error(qF3_coef) < tol );
+
+                  MixedBilinearForm blfw(&fespace_rt, &fespace_nd);
+                  blfw.AddDomainIntegrator(new VectorFEMassIntegrator(q3_coef));
+                  blfw.Assemble();
+                  blfw.Finalize();
+
+                  SparseMatrix * blfT = Transpose(blfw.SpMat());
+                  SparseMatrix * diff = Add(1.0,blf.SpMat(),-1.0,*blfT);
+
+                  REQUIRE( diff->MaxNorm() < tol );
+
+                  delete blfT;
+                  delete diff;
+               }
                SECTION("With Diagonal Matrix Coefficient")
```

Comment:
Would it be possible to add the VectorFE versions for Diagonal Matrix Coefficient tests?

Reply:
Yes, of course.  Thanks for catching this!  I think I found four such cases and added new tests for each.